### PR TITLE
fix template dot-files

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
The `cp $SRC/$d/*` does not copy dot files.

CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: templates/BUILD.bazel [hunk 1]
```diff
@@ -68,10 +68,13 @@ genrule(
         for d in skeleton empty-skeleton create-daml-app quickstart-scala quickstart-java; do
             mkdir -p $$OUT/$$d
             cp -rL $$SRC/$$d/* $$OUT/$$d/
-            # use default .gitignore and .dlint.yaml if they don't exist in the template
-            cp -n $$SRC/default-gitattributes $$OUT/$$d/.gitattributes
-            cp -n $$SRC/default-gitignore $$OUT/$$d/.gitignore
-            cp -n $$SRC/default-dlint-yaml $$OUT/$$d/.dlint.yaml
+            for f in gitattributes gitignore dlint.yaml; do
+                if [ -f "$$SRC/$$d/.$$f" ]; then
```
Comment (cocreature) [inline templates/BUILD.bazel:72]: I don’t get this part, when do we hit this?

Comment (cocreature) [reply templates/BUILD.bazel:72]: Are there templates that have custom files to override these?

Comment (garyverhaegen-da) [reply templates/BUILD.bazel:72]: `create-daml-app` has a custom [`.gitignore`](https://github.com/digital-asset/daml/blob/main/templates/create-daml-app/.gitignore) and [.dlint.yaml](https://github.com/digital-asset/daml/blob/main/templates/create-daml-app/.dlint.yaml).

I'm not aware of a template with a custom `.gitattributes` at this point, but I did not see a strong reason to make a special case out of it.

Comment (cocreature) [reply templates/BUILD.bazel:72]: makes sense, thanks

**Review by cocreature — APPROVED** [review]

**Pull request merged**